src/libpriv/rpmostree-scripts.cxx

```diff
@@ -373,8 +373,6 @@ rpmostree_run_script_in_bwrap_container (int rootfs_fd, GLnxTmpDir *var_lib_rpm_
   if (bridge_sysusers != NULL)
     bwrap->setenv ("RPMOSTREE_EXP_BRIDGE_SYSUSERS", rust::String (bridge_sysusers));
 
-  bwrap->setenv ("RPMOSTREE_SCRIPT_PKG_NAME", name);
```

Comment:
This comes with some negative side-effects, as you will start having conflicts/fights on the same filepath between different packages that try to deal with the same users/groups names.

Reply:
They already will fight at runtime though, right?

Reply:
Not really, or at least not in the same way. `systemd-sysusers` knows how to de-duplicate entries: it will happily multiple matching definitions, while in cases of mismatched will pick the highest sorting one and ignore+warn about conflicts.

But I was more focused about the filesystem side honestly. When installing packages, can we end up committing these filepaths to separate ostree commits? Can we end up with conflicts at checkout time then?

Reply:
I don't fully agree with the reasoning, but as you prefer.

Reply:
Well, let's keep the discussion going!  I don't want to dismiss your concern.
(But, this does fix what is far the biggest issue I've hit in trying Dockerfile builds)

Can you file an issue for your concern?  Do you just want to try to go back to per-package files?

Reply:
No worries. I agree this isn't a blocker, the concern was on dropping the package name from the filepath (i.e. blending artifacts from different packages).
We already had a bit of discussion on this at https://github.com/coreos/rpm-ostree/pull/3778#discussion_r930434033.
The longer-form filename seems a safer design with regards to keeping package content isolated.
But maybe this shorter form works well enough in practice, so let's see if we actually hit real bugs before revisiting this discussion once again.